docs/get-started/getting-started-gcp.md

```diff
@@ -0,0 +1,113 @@
+* Four NVIDIA T4 to each worker nodes
+* [Local SSDs](https://cloud.google.com/dataproc/docs/concepts/compute/dataproc-local-ssds) is recommended to improve IO for Spark scratch places  
+* Component gateway enabled for accessing Web UIs hosted on the cluster
+* Configuration for [GPU scheduling and isolation](/get-started/yarn-gpu.html)
+
+
+```bash
+    export REGION=[Your Prefer GCP Region]
+    export GCS_BUCKET=[Your GCS Bucket]
+    export CLUSTER_NAME=[Your Cluster Name]
+    export NUM_GPUS=4
+    export NUM_WORKERS=5
+
+gcloud dataproc clusters create $CLUSTER_NAME  \
+    --region $REGION \
+    --image-version=preview-ubuntu \
+    --master-machine-type n1-standard-16 \
+    --num-workers $NUM_WORKERS \
+    --worker-accelerator type=nvidia-tesla-t4,count=$NUM_GPUS \
+    --worker-machine-type n1-highmem-32\
+    --num-worker-local-ssds 4 \
+    --initialization-actions gs://dataproc-initialization-actions/gpu/install_gpu_driver.sh,gs://dataproc-initialization-actions/rapids/rapids.sh \
+    --optional-components=ANACONDA,JUPYTER,ZEPPELIN \
+    --metadata gpu-driver-provider="NVIDIA" \
+    --metadata rapids-runtime=SPARK \
+    --bucket $GCS_BUCKET \
+    --enable-component-gateway \
+    --properties="^#^spark:spark.yarn.unmanagedAM.enabled=false#spark:spark.task.resource.gpu.amount=0.125#spark:spark.executor.cores=8#spark:spark.task.cpus=1#spark:spark.executor.memory=8G"`
+``` 
+This may take around 5-15 minutes to complete. You can navigate to Dataproc clusters tab in the Google Cloud Console to see the progress.
+
+![Dataproc Cluster](../img/dataproc-cluster.png)
+
+## Run Pyspark and Scala Notebook a Dataproc Cluster Accelerated by GPU
+To use notebooks with Dataproc cluster, click on the cluster name under Dataproc cluster tab and navigate to the "Web Interfaces" Tab. Under the "Web Interfaces", click on JupyterLab or Jupyter link to start to use sample [Mortgage ETL on GPU Jupyter Notebook](../demo/GCP/Mortgage-ETL-GPU.ipynb) to process full 17 years [Mortgage data](https://rapidsai.github.io/demos/datasets/mortgage-data). 
+
+![Dataproc Web Interfaces](../img/dataproc-service.png)
+
+The notebook will first transcode CSV files into Parquet Files and then run a ETL query to prepare the dataset for Training. In the sample notebook, we use 2016 data as evaluation set and the rest as training set, saving to respective GCS location. 
+First stage with default configuration in notebook should take ~110 seconds (1/3 of CPU execution time with same config) whereas second stage takes ~170 seconds (1/6 of CPU execution time with same config).
+Once data is prepared, we use [Mortgage XGBoost4j Scala Notebook](../demo/GCP/mortgage-xgboost4j-gpu-scala.zpln) in Dataproc Zeppelin service to execute the training job on GPU. Since GITHUB cannot render zeppelin notebook, we prepared a [Jupyter Notebook with Scala code](../demo/GCP/mortgage-xgboost4j-gpu-scala.ipynb) for you to view code content. 
```

Comment:
Why use Zepplin if we have a Jupyter notebook that does the same thing? Maintaning 2 is a lot more problematic than maintaining 1. What is more because of how the notebooks work we are actually maintaining a lot of these notebooks, and it feels really frustratingly needless.

What is more the Jupyter note book does not render for me in github either. I just get a spinner until it gives up and returns an error message.
```
Sorry, something went wrong. Reload?
```

Reply:
Sometime GITHUB has issue render notebooks. We do need to figure out a way to test notebooks. If we define the spark context in the notebook, a way to test is: 
```jupyter nbconvert --to script /tmp/test.ipynb --output /tmp/test
pyspark /tmp/test.py || exit 1
```

Could we merge the notebook for the dataproc and implement test later?

Reply:
I filed a follow-on issue here: https://github.com/NVIDIA/spark-rapids/issues/367